cmd/main.go

```diff
@@ -434,7 +436,35 @@ func autoMigrationWithLeader(migrationLeader leader.ElectorInterface, db *gorm.D
 }
 
 func uploadBaseISOWithLeader(uploadLeader leader.ElectorInterface, objectHandler s3wrapper.API, generator generator.ISOInstallConfigGenerator, log logrus.FieldLogger) error {
```

Comment:
how does all this code tested? in case someone will make changes.

Reply:
I think pushing this into a different package and making it unit-testable is the right approach. To avoid conflicts with @avishayt I'll skip this item, _unless_ addressing this comment (https://github.com/openshift/assisted-service/pull/645#discussion_r523721694) forces my hand.